### ISSUE TITLE
Psi4 changes directory if DFT-D3 is used

### ISSUE THREAD

**Issue by raimis** [psi4/psi4#1064]
Psi4 changes directory if **DFT-D3** is used:
```python
import os
import psi4

print(psi4.__version__)

print(os.getcwd())

psi4.set_output_file('/dev/null')
psi4.geometry('He 0 0 0')
psi4.energy('PBE-D3/3-21G')

print(os.getcwd())
```
```bash
$ python test1.py 
1.2rc3
/shared/raimis/debug/psi4
/home/tmp/psi.15136..dftd3.a9f60419
```
This does not happen if **DFT-D2** is used:
```python
import os
import psi4

print(psi4.__version__)

print(os.getcwd())

psi4.set_output_file('/dev/null')
psi4.geometry('He 0 0 0')
psi4.energy('PBE-D2/3-21G')

print(os.getcwd())
```
```bash
$ python test2.py 
1.2rc3
/shared/raimis/debug/psi4
/shared/raimis/debug/psi4
```
A small detail, but it is messing up our pipelines. And it is similar to #817.

**Comment by loriab** [comment]
Thanks, I’ll take a look and fix it before 1.2. Will probably be Monday though as at a conf

**Comment by loriab** [comment]
And it’s not surprising that d2 and d3 behave differently, as former is usually run internally and latter through dftd3 exe
